system/argtable3/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config ESPAPPS_ARGTABLE3
+	bool "ARGTABLE3 library"
+	default n
+	---help---
+		Enables the ARGTABLE3 library.
+
+if ESPAPPS_ARGTABLE3
+
+config ESPAPPS_ARGTABLE3_VERSION
```

Comment:
ESPAPPS_ to SYSTEM_? other looks good to me, thanks.

Reply:
Done